tests/testData.h

```diff
@@ -703,6 +705,46 @@ const Species &benzeneSpecies()
     return benzene_;
 }
 
+// Return Node-Graph water test species
+Species *createWater(Graph *parentGraph)
+{
+    const auto name = "Water";
+
+    // Add water species node
+    auto speciesNode = std::make_unique<SpeciesNode>(parentGraph);
+    auto species = &(speciesNode.get()->species());
+    parentGraph->addNode(std::move(speciesNode), name);
+
+    // Set up water species and atom types
+    species->clear();
+    species->setName(name);
+
+    auto oW = std::make_shared<AtomType>("OW");
+    oW->setZ(Elements::Element::O);
+    species->addAtom(Elements::Element::O, Vector3{0.156743, -0.152502, 0.317676}, -0.82, oW);
+
+    auto hW1 = std::make_shared<AtomType>("HW");
+    hW1->setZ(Elements::Element::H);
+    species->addAtom(Elements::Element::H, Vector3{-0.257130, 0.637822, 0.198948}, 0.41, hW1);
+
+    auto hW2 = std::make_shared<AtomType>("HW");
+    hW2->setZ(Elements::Element::H);
+    species->addAtom(Elements::Element::H, Vector3{0.100387, -0.485320, -0.516624}, 0.41, hW2);
+
+    species->simplifyAtomTypes();
```

Comment:
I'd suggest we clean this up and use a bit of trig (as in the function below). Also, single atom type for both H to start with:

```suggestion
    auto oW = std::make_shared<AtomType>(Elements::Element::O, "OW");
    species->addAtom(Elements::Element::O, {}, -0.82, oW);

    auto hW = std::make_shared<AtomType>(Elements::Element::H, "HW");
    species->addAtom(Elements::Element::H, {1, 0, 0}, 0.41, hW);
    species->addAtom(Elements::Element::H, {cos(DissolveMath::toRadians(113.24)), sin(DissolveMath::toRadians(113.24)), 0.0},
                     0.41, hW);
```